drivers/storage/portworx/component/portworx_basic.go

```diff
@@ -379,6 +379,11 @@ func (c *portworxBasic) createClusterRole() error {
 					Resources: []string{"*"},
 					Verbs:     []string{"*"},
 				},
+				{
+					APIGroups: []string{"portworx.io"},
+					Resources: []string{"portworxdiags", "portworxdiags/status"},
+					Verbs:     []string{"*"},
```

Comment:
I think we should explicitly mention the list of verbs instead of using wildcard here

Reply:
Can do